docs/software/markdown-migration.md

```diff
@@ -13,8 +13,8 @@ Initial conversion with Pandoc
 
 [Pandoc](http://pandoc.org/) is a tool that's useful for automated conversion of markdown languages. [Once installed](http://pandoc.org/installing.html) (alternatively, run pandoc [via docker](#using-docker)), run the following command to convert TWiki to Markdown:
 
-```bash
-pandoc -f twiki -t markdown_github <INPUT FILE> > <OUTPUT FILE>
+```console
+$ pandoc -f twiki -t markdown_github <INPUT FILE> > <OUTPUT FILE>
```

Comment:
This looks worse than before, with `INPUT`/`OUTPUT` and `FILE` having different font faces